aperturedb/ConnectionDataCSV.py

```diff
@@ -66,7 +66,8 @@ def __init__(self, filename):
         self.src_class   = self.header[1].split("@")[0]
         self.src_key     = self.header[1].split("@")[1]
         self.dst_class   = self.header[2].split("@")[0]
-        self.dst_key     = self.header[2].split("@")[1]
+        # Pandas appends a .n to the column name if there is a duplicate
```

Comment:
Ah i see. So, let's say you are trying to connect "Persons", who are friends, based on their uids, the csv would have 2 headers which read the same, i.e Person@uid and Person@uid.

when pandas reads this csv, to avoid this conflict, it implicitly renames conflicting columns as col.0, col.1,col.2 ....

That breaks the queries we make.

Reply:
ah nice, hadn't thought of that